ReStyle-Frontend/src/index.html

```diff
@@ -1,6 +1,8 @@
 <!doctype html>
 <html lang="en">
 <head>
+  <link type="text/css" rel="stylesheet" href="https://cdn.firebase.com/libs/firebaseui/3.0.0/firebaseui.css" />
```

Comment:
This should be changed eventually. See https://www.npmjs.com/package/firebaseui-angular#add-the-firebaseui-css-to-your-imports
For now this is fine

Reply:
A comment should be added here, though